<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="fisrtStageTable"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="uploadBtn">
        <property name="text">
         <string>Загрузить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createBtn">
        <property name="text">
         <string>Сгененировать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="uploadToExcel"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="uploadToExcel">
   <property name="text">
    <string>Выгрузить в excel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
